week_3/Binary_Tree/invert_binary_tree/invert_binary_tree.test.js

```diff
@@ -0,0 +1,16 @@
+const invertTree = require("./invert_binary_tree");
+const BinarySearchTree = require("../index");
+
+test("invert_binary_tree accurately inverts the given binary tree", () => {
+	let bst = new BinarySearchTree();
+	bst.insert(10);
+	bst.insert(15);
+	bst.insert(6);
+	bst.insert(3);
+	bst.insert(8);
+	bst.insert(20);
+	invertTree(bst);
+	expect(JSON.stringify(invertTree(bst.root))).toBe(
+		'{"val":10,"left":{"val":15,"left":{"val":20,"left":null,"right":null},"right":null},"right":{"val":6,"left":{"val":8,"left":null,"right":null},"right":{"val":3,"left":null,"right":null}}}'
```

Comment:
You may want to install Prettier in your IDE to format this to be more legible. It's a bit hard to understand the data as its formatted atm

Reply:
I improved readability significantly by not stringifying JSON and instead checking equality with the object itself.

This is much better because in order for JSON.stringify to work, the strings had to match exactly, whereas here the object can be arranged in a much more readable format.